pkg/cmd/list.go

```diff
@@ -326,7 +329,7 @@ func (w *WhoCan) Check(action Action) (roleBindings []rbac.RoleBinding, clusterR
 
 // CheckAPIAccess checks whether the subject in the current context has enough privileges to query Kubernetes API
 // server to perform Check.
-func (w *WhoCan) CheckAPIAccess(action Action) ([]string, error) {
+func (w *WhoCan) CheckAPIAccess(ctx context.Context, action Action, opts metav1.CreateOptions) ([]string, error) {
```

Comment:
I would not change the interface of CheckAPIAccess method. Just pass context.Background() and &metav1.CreateOptions{} to the client